offchain/Cargo.toml

```diff
@@ -91,5 +92,3 @@ urlencoding = "2.1"
 users = "0.11"
 uuid = "1.4"
 
-[profile.release]
-strip = true  # Automatically strip symbols from the binary.
```

Comment:
Was this on purpose?

Reply:
No. Got it from merge/rebase . Reverting